### PR TITLE
trad chinese localisation

### PR DESCRIPTION
**Pull request by lesorubshayan**

### CODE REVIEW

File: runtime/lang/menu_zh_tw.utf-8.vim [hunk 1]
```diff
@@ -149,7 +149,7 @@ menutrans &Folding			覆疊(Fold)設定(&F)
     menutrans &Delete\ Fold<Tab>zd		    刪除\ Fold(&D)<Tab>zd
     menutrans Delete\ &All\ Folds<Tab>zD	    刪除所有\ Fold(&A)<Tab>zD
     " ------------------------------------------------------------------------
-    menutrans Fold\ column\ &width		    設定\ Fold欄寬(&W)
+    menutrans Fold\ col&umn\ width		    設定\ Fold欄寬(&W)
```
Comment (zeertzjq) [inline runtime/lang/menu_zh_tw.utf-8.vim]:
```suggestion
    menutrans Fold\ col&umn\ width		    設定\ Fold欄寬(&U)
```

File: runtime/lang/menu_zh_tw.utf-8.vim [hunk 2]
```diff
@@ -171,7 +171,7 @@ menutrans Error\ &Window		錯誤訊息視窗(&W)
     menutrans &Open<Tab>:copen		開啟(&O)<Tab>:copen
     menutrans &Close<Tab>:cclose	關閉(&C)<Tab>:cclose
     " }}}
-menutrans &Set\ Compiler		設定編譯器Compiler(&S)
+menutrans Se&T\ Compiler		設定編譯器Compiler(&S)
```
Comment (zeertzjq) [inline runtime/lang/menu_zh_tw.utf-8.vim]:
```suggestion
menutrans Se&T\ Compiler		設定編譯器Compiler(&T)
```

File: runtime/lang/menu_zh_tw.utf-8.vim [hunk 3]
```diff
@@ -285,6 +285,32 @@ if has("toolbar")
...
 " }}}
+menutrans Open\ &Tab\.\.\.<Tab>:tabnew	打開選項卡\.\.\.<Tab>:tabnew
+menutrans Startup\ &Settings		啟動設定
+menutrans Toggle\ relati&ve\ Line\ Numbering<Tab>:set\ rnu!	切換相對行編號<Tab>:set\ nru!
+menutrans Toggle\ Line\ &Wrapping<Tab>:set\ wrap!		切換換行<Tab>:set\ wrap!
+menutrans Toggle\ W&rapping\ at\ word<Tab>:set\ lbr!		切換在單字處換行<Tab>:set\ lbr!
+menutrans Toggle\ Tab\ &Expanding<Tab>:set\ et!			切換選項卡展開<Tab>:set\ et!
+menutrans Toggle\ &Auto\ Indenting<Tab>:set\ ai!		切換自動縮排<Tab>:set\ ai!
+menutrans Toggle\ &C-Style\ Indenting<Tab>:set\ cin!		切換C\ 風格縮排<Tab>:set\ cin!
+menutrans &Spelling						拼字
```
Comment (zeertzjq) [inline runtime/lang/menu_zh_tw.utf-8.vim]:
```suggestion
menutrans Open\ &Tab\.\.\.<Tab>:tabnew	打開選項卡(&T)\.\.\.<Tab>:tabnew
menutrans Startup\ &Settings		啟動設定(&S)
menutrans Toggle\ relati&ve\ Line\ Numbering<Tab>:set\ rnu!	切換相對行編號(&V)<Tab>:set\ nru!
menutrans Toggle\ Line\ &Wrapping<Tab>:set\ wrap!		切換換行(&V)<Tab>:set\ wrap!
menutrans Toggle\ W&rapping\ at\ word<Tab>:set\ lbr!		切換在單字處換行(&R)<Tab>:set\ lbr!
menutrans Toggle\ Tab\ &Expanding<Tab>:set\ et!			切換選項卡展開(&E)<Tab>:set\ et!
menutrans Toggle\ &Auto\ Indenting<Tab>:set\ ai!		切換自動縮排(&A)<Tab>:set\ ai!
menutrans Toggle\ &C-Style\ Indenting<Tab>:set\ cin!		切換C\ 風格縮排(&C)<Tab>:set\ cin!
menutrans &Spelling						拼字(&S)
```

File: runtime/lang/menu_zh_tw.utf-8.vim [hunk 4]
```diff
@@ -285,6 +285,32 @@ if has("toolbar")
...
+">>>---------------- Tools/Spelling
+menutrans &Spell\ Check\ On			拼字檢查打開
+menutrans Spell\ Check\ &Off			拼字檢查關閉
+menutrans To\ &Next\ error<Tab>]s		到下一個錯誤<Tab>]s
+menutrans To\ &Previous\ error<Tab>[s		到上一個錯誤<Tab>[s
+menutrans Suggest\ &Corrections<Tab>z=		建議更正<Tab>z=
+menutrans &Repeat\ correction<Tab>:spellrepall	重複修正<Tab>:spellrepall
```
Comment (zeertzjq) [inline runtime/lang/menu_zh_tw.utf-8.vim]:
```suggestion
menutrans &Spell\ Check\ On			拼字檢查打開(&S)
menutrans Spell\ Check\ &Off			拼字檢查關閉(&O)
menutrans To\ &Next\ error<Tab>]s		到下一個錯誤(&N)<Tab>]s
menutrans To\ &Previous\ error<Tab>[s		到上一個錯誤(&P)<Tab>[s
menutrans Suggest\ &Corrections<Tab>z=		更正建議(&C)<Tab>z=
menutrans &Repeat\ correction<Tab>:spellrepall	重複更正(&R)<Tab>:spellrepall
```

File: runtime/lang/menu_zh_tw.utf-8.vim [hunk 5]
```diff
@@ -285,6 +285,32 @@ if has("toolbar")
...
+"-------------------
+menutrans Set\ language\ to\ "en"		將語言設定為\ "en"
+menutrans Set\ language\ to\ "en_au"		將語言設定為\ "en_au"
+menutrans Set\ language\ to\ "en_ca"		將語言設定為\ "en_ca"
+menutrans Set\ language\ to\ "en_gb"		將語言設定為\ "en_gb"
+menutrans Set\ language\ to\ "en_nz"		將語言設定為\ "en_nz"
+menutrans Set\ language\ to\ "en_us"		將語言設定為\ "en_us"
+menutrans &Find\ More\ Languages		尋找更多語言
```
Comment (zeertzjq) [inline runtime/lang/menu_zh_tw.utf-8.vim]:
```suggestion
menutrans &Find\ More\ Languages		尋找更多語言(&F)
```

File: runtime/lang/menu_zh_tw.utf-8.vim [hunk 6]
```diff
@@ -285,6 +285,32 @@ if has("toolbar")
...
+let g:menutrans_set_lang_to =			'將語言設定為'
+menutrans &Show\ File\ Types\ in\ menu		在選單中顯示文件類型
```
Comment (zeertzjq) [inline runtime/lang/menu_zh_tw.utf-8.vim]:
```suggestion
menutrans &Show\ File\ Types\ in\ menu		在選單中顯示文件類型(&S)
```

**Comment by zeertzjq** [comment]
See also #13031

**Comment by chrisbra** [comment]
waiting on ACK by @zeertzjq

File: runtime/lang/menu_zh_tw.utf-8.vim [hunk 7]
```diff
@@ -285,6 +285,32 @@ if has("toolbar")
   endfun
 endif
 " }}}
+menutrans Open\ &Tab\.\.\.<Tab>:tabnew	打開選項卡(&T)\.\.\.<Tab>:tabnew
+menutrans Startup\ &Settings		啟動設定(&S)
+menutrans Toggle\ relati&ve\ Line\ Numbering<Tab>:set\ rnu!	切換相對行編號(&V)<Tab>:set\ nru!
+menutrans Toggle\ Line\ &Wrapping<Tab>:set\ wrap!		切換換行(&V)<Tab>:set\ wrap!
```
Comment (zeertzjq) [inline runtime/lang/menu_zh_tw.utf-8.vim]: Oops
```suggestion
menutrans Toggle\ Line\ &Wrapping<Tab>:set\ wrap!		切換換行(&W)<Tab>:set\ wrap!
```

**Comment by zeertzjq** [comment]
I'm not entirely sure about this. cc @ychin

**Comment by chrisbra** [comment]
closing

**Pull request closed without merging**